skl2onnx/__init__.py

```diff
@@ -7,7 +7,7 @@
 """
 Main entry point to the converter from the *scikit-learn* to *onnx*.
 """
-__version__ = "1.4.3"
+__version__ = "1.4.4.2"
```

Comment:
Most python package version are all three parts, why do you want to suggest a different form?

Reply:
Sorry for that, I tried a couple of times to fix the rendering of README.md on pypi. I had to convert markdown into rst with pypandoc.

Reply:
I'll close the issue. The package must be uploaded with twine.